workers/data_refinery_workers/processors/qn_reference.R

```diff
@@ -0,0 +1,59 @@
+###
+# qn_reference
+# Alex's Lemonad Stand Foundation
+# Childhood Cancer Data Lab
+#
+###
+
+#######################
+# The command interface!
+#######################
+
+library("optparse")
+library(data.table)
+library("preprocessCore")
+
+option_list = list(
+  make_option(c("-i", "--inputFile"), type="character", default="", 
+              help="inputFile", metavar="character"),
+  make_option(c("-o", "--outputFile"), type="character", default="", 
+              help="outputFile", metavar="character")
+); 
+
+opt_parser = OptionParser(option_list=option_list);
+opt = parse_args(opt_parser);
+
+filePath <- opt$inputFile
+outFilePath <- opt$outputFile
+
+# Get and use this DB
+message("Loading...")
+source("http://bioconductor.org/biocLite.R")
+
+# Read the data file
+message("Reading data file...")
+suppressWarnings(data <- fread(filePath, 
+					stringsAsFactors=FALSE, 
+					sep="\t", header=TRUE, 
+					autostart=10, 
```

Comment:
This means we're skipping the first 10, is that correct? Is that because of how the data is getting passed to this script in our system?

Reply:
Ah, no it does not mean that. Was looking at newer documentation where it says it's depreciated.